www/js/app.js

```diff
@@ -965,30 +993,34 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         }
     }
 
-  
     /**
      * Display the list of articles with the given array of DirEntry
-     * @param {Array} dirEntryArray The array of dirEntries returned from the binary search
+     * @param {Array} dirEntryArray The array of dirEntries returned from the binary 
```

Comment:
Add missing word "search" back to end of comment.

Reply:
Nice, would be good as well to write in the PR descripion which tickets exactly will closed by tis PR.